src/Agent/NewRelic/Agent/Core/JsonConverters/LoadedModuleWireModelCollectionJsonConverter.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using NewRelic.Agent.Core.WireModels;
+using Newtonsoft.Json;
+
+namespace NewRelic.Agent.Core.JsonConverters
+{
+    public class LoadedModuleWireModelCollectionJsonConverter : JsonConverter<LoadedModuleWireModelCollection>
+    {
+        private const string JarsName = "Jars";
```

Comment:
I'm not sure where the best place is for this, but we might want to document why we're sending a list of "jars" in .NET. :)

Reply:
Yeah, that is a quirk of the system.  It was meant for Java.  Once the VM team gets some time with the data we send, they will loop back around and let us know what is required and we can trim down what we send.  At the same time we can push for the team to update the endpoint to have an "Assemblies" or similar label for us,

I'll add a note.

Reply:
Updated.